docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3.1'
 
 services:
     postgres:
-        image: postgres:10.7-alpine
+        image: postgres:13.7-alpine
```

Comment:
Not strictly necessary w/ this PR (initially bumped for postgres' UUID() type), but does bring us closer in line w/ production DB.

Reply:
I suppose it would be nice for this to be kept in sync with https://github.com/guardian/typerighter/blob/ffd616566e16c3e2fc0f00bafc6ceecd35b33563/cdk/lib/index.ts#LL281C40-L281C46